pointpats/distance_statistics.py

```diff
@@ -132,6 +132,7 @@ def f(
     metric="euclidean",
     hull=None,
     edge_correction=None,
+    seed=None,
```

Comment:
We should not use `seed` anymore but `rng` see SPEC7 - https://scientific-python.org/specs/spec-0007/